client/state/comments/reducer.js

```diff
@@ -126,7 +127,24 @@ export function totalCommentsCount( state = {}, action ) {
 	return state;
 }
 
+/**
+ * Keeps track of if a comment list request is in flight
+ * @param {object} state  - global redux state
+ * @param {object} action - redux action
+ * @returns {boolean} new redux state
+ */
+export function isCommentListLoading( state = false, action ) {
+	switch ( action.type ) {
+		case COMMENTS_LIST_REQUEST:
+			return true;
+		case COMMENTS_RECEIVE: //we may need a more specific success/error action here
+			return false;
+	}
```

Comment:
@dmsnell I'm not sure I follow here. Why would the `COMMENTS_LIST_REQUEST` action have related error data? Unless you meant something else, this still has the same issues as the counter example.

>  I'd be strongly opposed to adding a new SUCCESS and _ERROR

I understand your feelings on this 😄, but currently I don't see a good alternative here.